app/mailers/jobseeker_mailer.rb

```diff
@@ -24,4 +24,15 @@ def reset_password_instructions(record, token, _opts = {})
       subject: I18n.t("jobseeker_mailer.reset_password_instructions.subject"),
     )
   end
+
+  def unlock_instructions(record, token, _opts = {})
+    @jobseeker = record
+    @token = token
+
+    view_mail(
+      NOTIFY_JOBSEEKER_LOCKED_ACCOUNT_TEMPLATE,
+      to: @jobseeker.email,
+      subject: I18n.t("jobseeker_mailer.unlock_instructions.subject"),
```

Comment:
You can use the short lookup here too, just use `t(".subject")` and put the translation in the right place